content/en/docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -110,38 +105,32 @@ of `Always`.
 
 1. Get a shell into the restarted Container:
 
-    ```shell
-    kubectl exec -it redis -- /bin/bash
-    ```
+   ```shell
+   kubectl exec -it redis -- /bin/bash
+   ```
 
 1. In your shell, go to `/data/redis`, and verify that `test-file` is still there.
-    ```shell
-    root@redis:/data/redis# cd /data/redis/
-    root@redis:/data/redis# ls
-    test-file
-    ```
-
-1. Delete the Pod that you created for this exercise:
 
-    ```shell
-    kubectl delete pod redis
-    ```
+   ```shell
```

Comment:
(aside) We should also fix this line: with the prompt included, the next lines (including the sample output) are not a valid shell program.

I can see that a fix here is different to the intent of this PR.